CONTRIBUTING_MODELS.md

```diff
@@ -0,0 +1,235 @@
+# Model Contribution Guide
+
+KerasNLP has a plethora of cutting-edge "backbone" model ranging from BERT and
+RoBERTa to FNet and DeBERTa. We are always looking for more models and are always
+open to contributions!
+
+In this guide, we will walk you through the steps one needs to take in order to contribute
+a backbone model. For illustration purposes, let's assume that you want to
```

Comment:
a backbone model -> a new pre-trained model to KerasNLP

(we are talking about more than just "backbones" below)